web/static/css/styles.css

```diff
@@ -0,0 +1,210 @@
+        }
+        &[aria-expanded="false"] {
+          > span.ca-gov-icon-caret-up {
+            display: none;
+          }
+          > span.ca-gov-icon-caret-down {
+            display: inline-block;
+          }
+        }
+      }
+      @media (width <= 770px) {
+        flex-direction: column !important;
+        > div.settings-links {
+          margin-left: 0 !important;
+          justify-content: flex-start;
+          > a {
+            margin-left: 0;
+            margin-right: 1rem;
+            font-size: 0.8rem;
+          }
+        }
+      }
+      @media (width <= 350px) {
+        > div.social-media-links {
+          flex-direction: column;
+        }
+      }
+    }
+  }
+  > div.section-default > div.branding > div.header-organization-banner {
+    display: flex;
+    margin-right: 0;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    > a > div.logo-assets {
+      > img.logo-img {
+        max-height: 60px;
+        height: 60px;
+        aspect-ratio: 18/5;
+      }
+      > div.logo-text {
+        > span.color-primary {
+          color: var(--color-p2, #046b99);
+        }
+        > span.logo-state {
+          font-size: 1.4rem;
+          font-weight: 700;
+          position: relative;
+          top: -5px;
+        }
+        > span.logo-dept {
+          font-size: 2.4rem;
+          height: 2.5rem;
+        }
+      }
+      @media (width <= 991px) {
+        > img.logo-img {
+          max-height: 50px;
+        }
+        > div.logo-text {
+          > span.logo-state {
+            font-size: 1.3rem;
+            top: -2px;
+          }
+          > span.logo-dept {
+            font-size: 1.8rem;
+            height: 2.2rem;
+          }
+        }
+      }
+    }
+    @media (width <= 575px) {
+      margin-right: 15%;
+      > a > div.logo-assets {
+        > div.logo-text {
+          > span.logo-dept {
+            font-size: 1.6rem;
+            height: 2.2rem;
+          }
+          > span.logo-state {
+            font-size: 1rem;
+            position: relative;
+            top: -2px;
+          }
+        }
+        > img.logo-img {
+          max-height: 50px;
+        }
+      }
+    }
+  }
+  > div.mobile-controls > div.main-nav-icons {
+    top: -5rem;
+    > button.toggle-menu {
+      margin-top: 1rem;
+    }
+  }
+  > div.navigation-search {
+    > div.nav-drawer {
+      padding-bottom: 0;
+    }
+    > div.search-container > div.d-block > a.drawer-logo-link {
+      outline-offset: -3px;
+      > img.drawer-logo {
+        width: 232px;
+        filter: brightness(0) invert(1);
+        aspect-ratio: 18/5;
+      }
+    }
+  }
+  @media (width >= 992px) {
+    > div.full-width-nav > div.featured-search {
+      top: -65px !important;
+      > div.container {
+        display: flex;
+        > form#Search {
+          margin-left: auto;
+        }
+      }
+    }
+  }
+}
+
+.bg-p2-dark {
+  background-color: var(--color-p2-dark, #035376) !important;
+}
+.bg-p2-darker {
```

Comment:
Appears to be unused

Reply:
@thekaveman those are utilities classes that common for ca.gov but i can add them later.